eos/saveddata/ship.py

```diff
@@ -52,6 +52,9 @@ def __init__(self, item, parent=None):
         self.__itemModifiedAttributes.original = dict(self.item.attributes)
         self.__itemModifiedAttributes.original.update(self.EXTRA_ATTRIBUTES)
         self.__itemModifiedAttributes.overrides = self.item.overrides
+        
+        if "maximumRangeCap" in self.__itemModifiedAttributes.original:
+            cappingAttrKeyCache["maxTargetRange"] = "maximumRangeCap"
```

Comment:
The range cap works fine, it just doesn't currently have any exception for carriers and supers. A few months ago something was broken and you changed the capping system to fix it, but the current code never changes the cap from the default.